lib/reporter.js

```diff
@@ -10,8 +10,10 @@ function masterEnd(context) {
     try {
       fs.writeFileSync(`${_context.config.get('output:reports')}/summary.json`, JSON.stringify(output, null, '\t'));
     } catch (err) {
-      console.error('something went wrong writing report file summary.json');
-      console.error(err);
+      console.error(`
+        Something went wrong writing report file ${_context.config.get('output:reports')}/summary.json. 
+        This can happen if no tests matched your input criteria.
+      `);
```

Comment:
Is this line removed `console.error(err);` because err is never going to be different than the one already outputted?

Reply:
fair question.. let me test before/after and see if console.error(err) needs to go back in.